site/react/useUser.md

```diff
@@ -20,7 +20,7 @@ head:
 
 # useUser
 
-The `useUser` hook returns the authenticated [`User`](accountkit.alchemy.com/resources/types.html#user) if the signer is authenticated.
+The `useUser` hook returns the authenticated [`User`](https://accountkit.alchemy.com/resources/types.html#user) if the signer is authenticated.
```

Comment:
This was making the build fail a dead link check.

Reply:
oh I just fixed this in main >.> think it just need to be `/resources/types.html#user` instead of the full path with domain

Reply:
Thanks! Rebased and took took out the change here. Can I get a re-review?